README.md

```diff
@@ -1,140 +1,75 @@
-.setSpanSinceStartThresholdResource(R.dimen.scaleSpanSinceStartThreshold);
-```
-and for thresholds that are not expressed in pixels:
-```
-androidGesturesManager.getRotateGestureDetector().setAngleThreshold(ROTATE_ANGLE_THRESHOLD);
+To use the Gestures library, include it in your app-level `build.gradle` file.
+
+```java
+// In the root build.gradle file
+repositories {
+    mavenCentral()
+}
 ```
 
-#### Velocity
-Each progressive gesture with its respective `#onEnd()` callback will provide `X velocity` and `Y velocity` of the gesture at the moment of pointers leaving the screen.
+```java
+// In the app build.gradle file
+dependencies {
+    implementation 'com.mapbox.mapboxsdk:mapbox-android-plugin-offline:0.1.0'
+}
+```
 
-#### Enable/disable and interrupt
-Every gesture detector can be enabled/disable at any point in time using `#setEnabled()` method.
+The library is published to Maven Central and nightly SNAPSHOTs are available on Sonatype:
 
-Additionally, every progressive gesture can be interrupted, which will force it to meet start conditions again in order to resume. Popular use case would be to increase gesture's threshold when other is detected:
+```java
+// In the root build.gradle file
+repositories {
+    mavenCentral()
+    maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
+}
 ```
-    @Override
-    public boolean onScaleBegin(StandardScaleGestureDetector detector) {
-      // forbid movement when scaling
-      androidGesturesManager.getMoveGestureDetector().setEnabled(false); // this interrupts a gesture as well
-    
-      // increase rotate angle threshold when scale is detected, then interrupt to force re-check
-      RotateGestureDetector rotateGestureDetector = androidGesturesManager.getRotateGestureDetector();
-      rotateGestureDetector.setAngleThreshold(ROTATION_THRESHOLD_WHEN_SCALING);
-      rotateGestureDetector.interrupt();
-
-      return true;
-    }
-        
-    @Override
-    public boolean onScale(StandardScaleGestureDetector detector) {
-      float scaleFactor = detector.getScaleFactor();
-      ...
-      ...
-      return true;
-    }
-    
-    @Override
-    public void onScaleEnd(StandardScaleGestureDetector detector) {
-      // revert thresholds values
-      RotateGestureDetector rotateGestureDetector = androidGesturesManager.getRotateGestureDetector();
-      rotateGestureDetector.setAngleThreshold(Constants.DEFAULT_ROTATE_ANGLE_THRESHOLD);
-    }
+```java
+// In the app build.gradle file
+dependencies {
+	implementation 'com.mapbox.mapboxsdk:mapbox-android-gestures:0.3.0-SNAPSHOT'
+}
 ```
 
-## Detectors
-With this library you will be able to recognize gestures using detectors provided by the Support Library and more.
+To run the [sample code](#sample-code) on a device or emulator, include your [developer access token](https://www.mapbox.com/help/define-access-token/) in `developer-config.xml` found in the project. 
```

Comment:
An access token is not required to run the `TestApp`, it's only required to run the Mapbox Activity.

Reply:
Fixed